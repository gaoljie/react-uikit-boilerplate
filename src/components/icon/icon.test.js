import React from "react";
import "jest-styled-components";
import { mount } from "enzyme";

import Icon from "./index";

const props = {
  name: "star",
  width: "2px"
};

describe("Test Circle", () => {
  it("Matches the snapshot", () => {
    const wrapper = mount(<Icon {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
