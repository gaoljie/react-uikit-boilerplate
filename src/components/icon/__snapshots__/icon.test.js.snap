// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Circle Matches the snapshot 1`] = `
.c0 {
  width: 2px;
  height: 2px;
}

<Icon
  name="star"
  width="2px"
>
  <Styled(styled.div)
    as="svg"
    height="2px"
    name="star"
    width="2px"
  >
    <StyledComponent
      as="svg"
      forwardedComponent={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bwzfXH",
            "isStatic": false,
            "lastClassName": "c0",
            "rules": Array [
              "
  ",
              [Function],
              "
",
            ],
          },
          "displayName": "Styled(styled.div)",
          "foldedComponentIds": Array [
            "sc-bdVaJa",
          ],
          "render": [Function],
          "styledComponentId": "sc-bwzfXH",
          "target": "div",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        }
      }
      forwardedRef={null}
      height="2px"
      name="star"
      width="2px"
    >
      <svg
        className="c0"
        height="2px"
        name="star"
        width="2px"
      >
        <use
          xlinkHref="#star"
        />
      </svg>
    </StyledComponent>
  </Styled(styled.div)>
</Icon>
`;
